### ISSUE TITLE
Re-instate at least test per every exposed view function in `Page` & utilities

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/the-trans-dimension#454]
## Description
When we upgraded to `elm-pages v3` our tests broke because they were rigged against the data models in `v2`.

We fixed the runner and put up example tests for view code in News and About page #446 - but we also deleted the other coverage.

We might want to reinstate at least some more and test our utilities etc.

## Pages without tests
- [ ] Event
- [ ] Index
- [ ] Join Us
- [ ] News Item
- [ ] Partners
- [ ] Partner
- [ ] Privacy

**Comment by kimadactyl** [comment]
For me I'd love to have some BDD cucumber-style tests (and for PlaceCal too). It would be great if what it's meant to do is written out in plain English as we keep seeing how much space there is for confusion in the code base and it can act as living documentation too. A lot of PlaceCal is quite subtle so I'd just love to see something we can put in front of people using it that doesn't need a high level understanding of Elm or Ruby to comprehend.

Unsure if that's a different ticket or this one!
